librz/bin/format/dex/dex.h

```diff
@@ -9,6 +9,10 @@
 #include <rz_util.h>
 #include <rz_bin.h>
 
+#define RZ_DEX_RELOC_TARGETS "reloc-targets"
+#define RZ_DEX_VIRT_ADDRESS  0x0100000000
```

Comment:
Why not have this just 0?

Reply:
because i do not know why, but on mapped phy regions it will fail.
like if i map X:phy to X:virt it will fail, but as long as the mem regions are not over each other even if part of 2 different worlds, it will work.

Reply:
and this is an issue with RzIO world, not dex.

Reply:
And more specifically, analysis will fail.

Reply:
That is strange though and needs investigation, since pie elf usually also map at 0 and those work.

Reply:
i guess we can push this in another PR